addons/website_livechat/tests/test_livechat_basic_flow.py

```diff
@@ -273,7 +273,7 @@ def test_channel_visitor_data(self):
                 ),
                 "website.visitor": [
                     {
-                        "country": self.env["ir.model.data"]._xmlid_to_res_id("base.be"),
```

Comment:
When seeing this diff, country was already flat in visitor actually. 😄

I guess what the PR does at the end is cleaning the models instead. More like part of https://www.odoo.com/odoo/1519/tasks/4675862/project.task/4675842/project.task/4675821